### ISSUE TITLE
grimblast's cursor is not themed

### ISSUE THREAD

**Issue by coffebar** [ndom91/rose-pine-hyprcursor#5]
### Describe the bug

When I take screenshot on Hyprland (using [grimblast](https://github.com/hyprwm/contrib)), 
```
grimblast --notify copysave area /tmp/filename.png
```
 cursor to select the screenshot area should be something like this:
 
 
![image](https://github.com/user-attachments/assets/85e194f7-df81-4862-aa82-768406403207)

Instead, I see a tiny default cursor, not from the theme.  

![image](https://github.com/user-attachments/assets/8cfa5c52-e70a-4778-acab-b5b9a74932da)

I didn't notice this in any other programs. It's possible bug of grim or grimblust, but please make sure. 

### Reproduction

in description 

### System Info

```Shell
Arch Linux, AUR
```

### Used Package Manager

n/a

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ndom91** [comment]
Unfortunately I can't reproduce this. I use `grimblast` as well and see the crosshair cursor as intended :thinking: 

Can you provide any more details?

**Comment by coffebar** [comment]
```
Hyprland, built from branch main at commit 04421063af2941c6e27e6dca2bdc2c387778a3a5  (config: add order rule for layers (7697)).
Date: Mon Sep 9 10:10:08 2024
Tag: v0.43.0-7-g04421063, commits: 5203

flags: (if any)


System Information:
System name: Linux
Node name: crab
Release: 6.10.10-arch1-1
Version: #1 SMP PREEMPT_DYNAMIC Thu, 12 Sep 2024 17:21:02 +0000


GPU information:
00:02.0 VGA compatible controller [0300]: Intel Corporation TigerLake-LP GT2 [Iris Xe Graphics] [8086:9a49] (rev 01) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo
```
### Installed packages: 
https://github.com/coffebar/dotfiles/blob/main/pkglist-intel.txt

I have these lines in hyprland config to make cursor theme work after reboot:
```
# gtk cursor and icon themes
exec-once = gsettings set org.gnome.desktop.interface cursor-theme BreezeX-RosePine-Linux
exec-once = gsettings set org.gnome.desktop.interface icon-theme 'bloom-classic'
exec-once = gsettings set org.gnome.desktop.interface cursor-size 32
```

**Comment by ndom91** [comment]
Do you have the separate `breezex-rosepine-linux` package installed? The "normal" cursor package and mine here are different. 

Mine is only for hyprland / hyprcusor specifically. 

Your invocation of the gsettings call will look for the "normal" [cursor package](https://github.com/ful1e5/BreezeX_Cursor)

**Comment by coffebar** [comment]
Yeah, I have [this line](https://github.com/coffebar/dotfiles/blob/d14d81b1e0b2810dd71b72e67fe4bce3884862b2/.config/hyprland/hyprland.conf#L1) in my config. Some of the apps are using that cursor, so I keep both packages.   

> Do you have the separate `breezex-rosepine-linux` package installed? The "normal" cursor package and mine here are different.
> 
> Mine is only for hyprland / hyprcusor specifically and can be set with the hyprland config:
> 
> ```
> HYPRCURSOR_THEME,rose-pine-hyprcursor
> ```
> 
> Or the `hyprctl set-cursor` cmd.
> 
> Your invocation of the gsettings call will look for the "normal" [cursor package](https://github.com/ful1e5/BreezeX_Cursor)
